### PR TITLE
feat(documenting/apiresource): added documentation for @apiResourceAdditional tag

### PR DESCRIPTION
**Pull request by 090809**
ref: https://github.com/knuckleswtf/scribe/pull/414

### CODE REVIEW

**Comment by shalvah** [comment]
Please also add the tag to the annotation reference. https://github.com/knuckleswtf/scribe-docs/blob/master/laravel/reference/20-annotations.md

**Comment by 090809** [comment]
@shalvah forgot about it, sorry.
Updated example code and added to annotations

File: laravel/documenting/60-responses.md [hunk 1]
```diff
@@ -280,6 +280,46 @@ public function listMoreUsers()
 }
 ```
 
+### Additional Data
+If your endpoint returns a data with additional fields of meta information, you can tell Scribe it by using `@apiResourceAdditional`.
```
Comment (shalvah) [inline laravel/documenting/60-responses.md]:
```suggestion
If your endpoint returns additional fields using the API resource's `additional()` method, you can indicate this with `@apiResourceAdditional`:
```

File: laravel/documenting/60-responses.md [hunk 2]
```diff
@@ -280,6 +280,46 @@ public function listMoreUsers()
...
+        return [ 'id' => $this->id ];
+    }
+}
+
+class UserController extends Controller
+{
+    /**
+     * @apiResource App\Http\Resources\UserResource
+     * @apiResourceModel App\Models\User
+     * @apiResourceAdditional title="User created" message="User successful created"
```
Comment (shalvah) [inline laravel/documenting/60-responses.md]:
```suggestion
     * @apiResourceAdditional status="success" message="User created successfully"
```

Comment (shalvah) [inline laravel/documenting/60-responses.md]:
```suggestion
     * @apiResourceAdditional result=success message="User created successfully"
```

File: laravel/documenting/60-responses.md [hunk 3]
```diff
@@ -280,6 +280,46 @@ public function listMoreUsers()
...
+     */
+    public function store($request): JsonResponse
+    {
+        // ... some store code ...
+        return UserResource::make($event)->additional([
+            'title' => __('User created'),
```
Comment (shalvah) [inline laravel/documenting/60-responses.md]:
```suggestion
            'result' => __('success'),
```

File: laravel/documenting/60-responses.md [hunk 4]
```diff
@@ -280,6 +280,46 @@ public function listMoreUsers()
...
+            'message' => __('User successful created'),
```
Comment (shalvah) [inline laravel/documenting/60-responses.md]:
```suggestion
            'message' => __('User created successfully'),
```

File: laravel/documenting/60-responses.md [hunk 5]
```diff
@@ -280,6 +280,46 @@ public function listMoreUsers()
...
+}
+```
+Produces output (with default `data`-wrapper):
+```json
+{
+  "data": {
+    "id": 1
+  },
+  "title": "User created",
+  "message": "User successful created"
```
Comment (shalvah) [inline laravel/documenting/60-responses.md]:
```suggestion
  "message": "User created successfully"
```

File: laravel/documenting/60-responses.md [hunk 6]
```diff
@@ -280,6 +280,46 @@ public function listMoreUsers()
...
+    }
+}
+```
+Produces output (with default `data`-wrapper):
+```json
+{
+  "data": {
+    "id": 1
+  },
+  "title": "User created",
```
Comment (shalvah) [inline laravel/documenting/60-responses.md]:
```suggestion
  "result": "success",
```

**Review by shalvah — APPROVED** [review]

**Comment by shalvah** [comment]
Thanks

**Pull request merged**